index.html

```diff
@@ -7189,7 +7189,51 @@ <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
         </p>
       </section>
     </section>
-
+    <section class="informative">
+    <h3>Language maps</h3>
+    The following is an example credentials that demonstrate multiple language and base directionality support.
+           <pre class="example nohighlight"
+             title="A sample credential that demonstrates multiple languages">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
+  "issuer": {
+    "id": "https://university.example/issuers/565049",
+    "name": "Example University",
+    "description": "A public university focusing on teaching examples."
+  },
+  "validFrom": "2015-05-10T12:30:00Z",
+  "name": "Example University Degree",
+  "description": "2015 Bachelor of Science and Arts Degree",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "ExampleBachelorDegree",
+      "name": [
+        {
+          "@value": "Bachelor of Science and Arts",
+          "@language": "English"
```

Comment:
Should this not be `en` or `en-us` or `en-uk` or...?
```suggestion
          "@language": "en-us"
```